typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/presentation.ts

```diff
@@ -274,29 +274,62 @@ function buildOutput(output: string[]): string {
 /**
  * Returns the graph for all the nodes.
  *
- * Will insert the `rootItem` by default as a start node an search for other start nodes
+ * Inserts nodes for the default Start and End workflow elements (not actual WF items).
+ * Will insert the `rootItem` (or first node, if not explicitly annotated)
+ * and any secondary start nodes (e.g. Default Error Handler) as targets of the default Start node.
+ * Secondary start nodes will be placed at the same level (column) as the default Start node.
  *
- * Will insert the default `end` node as well as the others require it but it's not an actual workflow item
+ * Remaining items are placed acording to their targets.
+ * @param {WorkflowDescriptor} workflow
+ * @returns {Graph}
+ * @throws Error when there are targets missing or misspelled or when a secondary start node is targeted.
  */
-function getGraph(workflow: WorkflowDescriptor) {
-	// start node
-	const startNodes = [`item${findItemByName(workflow.items, workflow.rootItem) || "1"}`];
-	const nodes = workflow.items.map((item, i) => {
-		switch (item.strategy.getCanvasType()) {
-			case "error-handler":
-				startNodes.push(`item${i + 1}`);
-				break;
-			default:
-				break;
-		}
+function getGraph(workflow: WorkflowDescriptor): Graph {
+	let errors = [];
+	const nodes: GraphNode[] = workflow.items.map((item, i) => getGraphNode(item, i, errors));
+	if (errors.length) {
+		errors.unshift("Failed to create graph nodes for:");
+	}
+	else try {
+		// default end node
+		nodes.push({ name: DEFAULT_END_ITEM_NAME, origName: "End0", targets: [] });
+		// default start node with target - root element
+		const startNode = {
+			name: Graph.START,
+			origName: "Start",
+			targets: [`item${findItemByName(workflow.items, workflow.rootItem) || "1"}`]
+		};
+		nodes.unshift(startNode);
+		// adding default error handler (if any) as target of the Default start node (secondary start node)
+		startNode.targets.push(...DefaultErrorHandlerDecoratorStrategy.getDefaultErrorHandlerNodes(workflow.items, nodes));
```

Comment:
Odd, It's not a target? But I guess you need it since you can't have more than one starts?

Reply:
Cannot be targeted by other items; start is not an item, it's added automatically to serve as tree root (also calculate the start node position)

Reply:
The default error handler can't be targeted by other items. The default Start node is not an item, however treating it as virtual tree root helps calculate the start position and tree structure more easily.